package.json

```diff
@@ -7,7 +7,7 @@
     "web": "expo start --web",
     "eject": "expo eject",
     "build": "tsc --project tsconfig.json",
-    "lint": "tslint --project tsconfig.json",
+    "lint": "eslint",
```

Comment:
probably want to pass in the --fix flag to have it attempt to fix issues automatically